doc/visual-programming/source/widgets/unsupervised/mds.md

```diff
@@ -57,6 +57,16 @@ widget. It is in many respects similar to the `Scatter Plot
 <../visualize/scatterplot>` widget, so we recommend reading that
 widget's description as well.
 
+Preprocessing
+-------------
+
+MDS uses default preprocessing if necessary. It executes it in the following order:
```

Comment:
Doesn't MDS only apply these preprocessors if the data was given directly, without going through distances? It that sis so, we should mention this.

Reply:
You're right, I will add it.